src/core/mod.rs

```diff
@@ -134,6 +136,14 @@ impl std::ops::AddAssign for Size {
     }
 }
 
+/// converts a `Key`s keystroke_buffer into a `String`
+pub fn keystrokes_to_string(keystroke_buffer: &Vec<Key>) -> String {
```

Comment:
This should be `&[Key]` instead of `&Vec<Key>` - it's slightly faster and more general.